packages/amplify_core/lib/src/logger/ansi_pretty_printer.dart

```diff
@@ -0,0 +1,85 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'package:amplify_core/amplify_core.dart';
+import 'package:ansicolor/ansicolor.dart';
+import 'package:logging/logging.dart';
+
+class AnsiPrettyPrinter implements AmplifyLoggerPlugin {
```

Comment:
I would remove this if we have questions about where it does and doesn't work.

Reply:
Agreed, we can always add it back later